method/group.js

```diff
@@ -93,6 +94,32 @@ module.exports = function GroupMethods (ssb, keystore, state) {
 
       ssb.publish(content, cb)
     },
+    listAuthors (groupId, cb) {
+      const query = [{
+        $filter: {
+          value: {
+            content: {
+              type: 'group/add-member'
+            }
+          }
+        }
+      }]
+
+      pull(
+        ssb.query.read({ query }),
+        pull.filter(addMemberSpec.isValid),
+        pull.filter(msg =>
+          groupId === msg.value.content.recps[0]
+        ),
```

Comment:
slightly faster to swap these two round

```suggestion
        pull.filter(msg =>
          groupId === msg.value.content.recps[0]
        ),
        pull.filter(addMemberSpec.isValid),
```

Reply:
yeah but then we'd have to add null checks